### PR TITLE
added logs to loadAttachment

### PR DESCRIPTION
**Pull request by rmcdole**

### CODE REVIEW

File: src/main/java/com/bullhorn/dataloader/service/csv/CsvFileWriter.java [hunk 1]
```diff
@@ -100,7 +94,8 @@ public synchronized void writeRow(JsonRow jsonRow, Result result) throws IOExcep
 
     public synchronized void writeAttachmentRow(String[] data, Result result) throws IOException {
```
Comment (ndickerson) [inline src/main/java/com/bullhorn/dataloader/service/csv/CsvFileWriter.java]: Suggest renaming this method to `writeRow` that differs from the `writeRow` above only by the first parameter.  Then, the above method can just call `writeRow(jsonRow.getValues())` and we can remove the redundant code.

File: src/main/java/com/bullhorn/dataloader/service/executor/EntityAttachmentConcurrencyService.java [hunk 2]
```diff
@@ -23,30 +27,41 @@
     private final CsvFileWriter csvWriter;
     private final PropertyFileUtil propertyFileUtil;
     private final BullhornData bullhornData;
+    private final Method method;
+    private final PrintUtil printUtil;
+    private ActionTotals actionTotals;
```
Comment (ndickerson) [inline src/main/java/com/bullhorn/dataloader/service/executor/EntityAttachmentConcurrencyService.java]: Add final keyword for consistency

File: src/main/java/com/bullhorn/dataloader/service/executor/EntityAttachmentConcurrencyService.java [hunk 3]
```diff
@@ -23,30 +27,41 @@
...
         this.csvWriter = csvWriter;
         this.executorService = executorService;
         this.propertyFileUtil = propertyFileUtil;
         this.bullhornData = bullhornData;
+        this.printUtil = printUtil;
+        this.actionTotals = actionTotals;
     }
 
-    public void runLoadAttchmentProcess() throws IOException {
+    public void runLoadAttchmentProcess() throws IOException, InterruptedException {
```
Comment (tkennedy0617) [inline src/main/java/com/bullhorn/dataloader/service/executor/EntityAttachmentConcurrencyService.java]: misspelled Attachment

File: src/main/java/com/bullhorn/dataloader/task/AbstractTask.java [hunk 4]
```diff
@@ -66,4 +76,31 @@ protected void getAndSetBullhornEntityInfo() {
...
+        if(rowProcessedCount.intValue() % 111 == 0) {
+            printUtil.printAndLog("Processed: " + NumberFormat.getNumberInstance(Locale.US).format(rowProcessedCount) + " records.");
+        }
+    }
+
+    private void updateActionTotals(Result result) {
+        if(result.getAction().equals(Result.Action.INSERT)) {
+            actionTotals.incrementTotalInsert();
+        } else if(result.getAction().equals(Result.Action.UPDATE)){
+            actionTotals.incrementTotalUpdate();
```
Comment (ndickerson) [inline src/main/java/com/bullhorn/dataloader/task/AbstractTask.java]: Please add the `else if` clause for the DELETE action that is in the DeleteTask, for completeness.

File: src/test/java/com/bullhorn/dataloader/task/LoadAttachmentTaskTest.java [hunk 5]
```diff
@@ -89,8 +96,8 @@ public void loadAttachmentSuccessTest() throws Exception {
     public void loadAttachmentFailureTest() throws ExecutionException, IOException {
         //arrange
         String[] expectedValues = {"1", "testResume/Test Resume.doc", "0"};
-        Result expectedResult = new Result(Result.Status.FAILURE, null, null, "Test");
-        task = new LoadAttachmentTask("Candidate", dataMap, csvFileWriter, propertyFileUtil, bullhornData);
+        Result expectedResult = new Result(Result.Status.FAILURE, Result.Action.NOT_SET, -1, "Test");
```
Comment (ndickerson) [inline src/test/java/com/bullhorn/dataloader/task/LoadAttachmentTaskTest.java]: Could benefit from replacing `new Result(Result.Status.FAILURE, null, null, "Test")` with: `Result.Failure("Test")`.

**Pull request merged**